stock_release_channel/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    stock_release_channel_id = fields.Many2one(
```

Comment:
@sbejaoui A partner could be in many release channels. Some can be asleep or closed or ...